cp-file-error.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+const NestedError = require('nested-error-stacks');
+
+class CpFileError extends NestedError {
+	constructor(message, nested) {
+		super(message, nested);
+		Object.assign(this, nested);
+	}
+}
+
+CpFileError.prototype.name = 'CpFileError';
```

Comment:
The correct way for ES2015 classes is to set `this.name` in the constructor instead.